src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.datasource.username=root
```

Comment:
Never provide login credentials within a repo :)
Please leave blank for sensitive data

Reply:
Pai asta nu e intr-un repo. Credentialele astea sunt puse intr-un fisier de configurare. Vrei sa le cripteze sau la ce te referi?